src/main.js

```diff
@@ -13,7 +13,7 @@ const getLogger = () => {
 			level: process.env.CONSOLE_LOG_LEVEL || 'silly'
 		});
 
-		if (process.env.NODE_ENV === 'production' && process.env.SPLUNK_HEC_TOKEN) {
+		if ((process.env.NODE_ENV === 'production' || process.env.LOG_TO_SPLUNK === 'true') && process.env.SPLUNK_HEC_TOKEN) {
```

Comment:
Could this safely be simplified down to checking for a `SPLUNK_HEC_TOKEN` variable?

We don't include it in development or on CircleCI, so likely yes.

Reply:
that was added in 3 years ago which is before I joined so I don't quite know what was happening then.